docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2059,7 +2084,7 @@ In this case, only requests to `example.org/script.js` will be "redirected" to `
 
 :::info Compatibility
 
-Rules with `$redirect-rule` modifier are not supported by AdGuard Content Blocker, AdGuard for iOS, and AdGuard for Safari.
+Rules with `$redirect-rule` modifier are not supported by AdGuard Content Blocker, AdGuard for iOS, AdGuard for Safari, and AdGuard Browser Extension MV3.
```

Comment:
@105th it makes sense to link the discussion in the W3C group.

Reply:
@SeverAnna https://github.com/w3c/webextensions/issues/493